CMakeLists.txt

```diff
@@ -149,17 +149,24 @@ endif()
 # NMODL specific options
 # =============================================================================
 if(CORENRN_ENABLE_NMODL)
-  find_package(nmodl REQUIRED)
-  include_directories(${nmodl_INCLUDE})
+  find_package(nmodl)
 
-  # copy to make nrnivmodl-core work from build as well instal directory
-  configure_file(${nmodl_BINARY} ${CMAKE_BINARY_DIR}/bin/nmodl COPYONLY)
-  set(CORENRN_NMODL_BINARY ${CMAKE_BINARY_DIR}/bin/nmodl)
+  if (nmodl_FOUND)
+    # copy to make nrnivmodl-core work from build as well install directory
+    configure_file(${nmodl_BINARY} ${CMAKE_BINARY_DIR}/bin/nmodl COPYONLY)
```

Comment:
I didn't see this before.. is this really necessary?
We really shouldn't have to copy around binaries to make things work...

Reply:
agree! fixed this as per our discussion.